RevoValidation/src/Rules.swift

```diff
@@ -37,5 +52,10 @@ public struct Rules : ExpressibleByStringLiteral {
         rules.map { $0.errorMessage }.map { NSLocalizedString($0, comment: $0) } .implode(" | ")
     }
     
+    public var shouldShowFirstErrorMessage:String {
+        guard rules.isEmpty == false else { return "" }
```

Comment:
si la funcio es diu should, vol dir que ha de tornar un Bool, si tu tornes missatges hauria de tenir un altre nom, estil, errorMessage o algu així